docs/guides/installation.rst

```diff
@@ -0,0 +1,33 @@
+Installation
+------------
+
+The document describes instructions using which you can install manticore from
+source.
+
+- Install the toolchain using: `./scripts/install-toolchain`
+
+After you have toolchain installed you can build Manticore using
+
+`make`
+
+The build system generates a `kernel.iso` image, can be launched under QEMU with:
+
+`$ ./scripts/run kernel.iso`
+
+Manticore Exploitation and Hacker's Guide
```

Comment:
I am not sure what you mean with "exploitation" here. I am using the term "hacking" in the original context of _programming_ (see the [Jargon file definition](http://www.catb.org/jargon/html/H/hacker.html), for example). There's no connection to exploiting security vulnerabilities in this document.